src/core/Header/HeaderTitle.tsx

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import '@itwin/itwinui-css/css/header.css';
+
+import cx from 'classnames';
+import React from 'react';
+
+import { useTheme } from '../utils/hooks/useTheme';
+
+export type HeaderTitleProps = {
+  /**
+   * Logo shown before the main content.
+   */
+  logo: JSX.Element;
+} & React.ButtonHTMLAttributes<HTMLDivElement>;
```

Comment:
`CommonProps` might be enough here, not sure we want all the native button props when we don't even use it

Reply:
Shouldnt be Button, it was with the previous implementation, for the same reason as the Header, but I'll change to `CommonProps`

Reply:
Changed to `CommonProps`